src/tools/rustfmt/src/patterns.rs

```diff
@@ -40,18 +40,18 @@ pub(crate) fn is_short_pattern(pat: &ast::Pat, pat_str: &str) -> bool {
 
 fn is_short_pattern_inner(pat: &ast::Pat) -> bool {
     match pat.kind {
-        ast::PatKind::Rest | ast::PatKind::Wild | ast::PatKind::Lit(_) => true,
```

Comment:
pls revert

Reply:
Well, this is a weird one: at first I didn't remove the `ast::`, and the checks failed with [a weird formatting error](https://github.com/rust-lang/rust/pull/118157#issuecomment-1822092363) despite my local `tidy` runs passing. I didn't feel like investigating, and the rest of the file doesn't use the `ast::` prefix for `PatKind` so I settled on this x)

If you insist I'll try again, and if the error persists I'll put the `Never` on its own match arm

Reply:
Welp I seem to have fixed it somehow, never mind then; I have reverted.